core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -60,12 +61,14 @@ var ExplorationEditorFeedbackTab = function() {
       `${suggestionRowClassName} ${feedbackSubjectClassName}`,
       suggestionDescription));
     expect(await matchingRow.isDisplayed()).toBe(true);
-    await matchingRow.click();
+    await action.click('Matching Row', matchingRow);
     expect(await viewSuggestionButton.isDisplayed()).toBe(true);
-    await viewSuggestionButton.click();
+    await action.click('View Suggestion Button', viewSuggestionButton);
     expect(await acceptSuggestionButton.isDisplayed()).toBe(true);
-    await suggestionCommitMessageInput.sendKeys('Commit message');
-    await acceptSuggestionButton.click();
+    await action.sendKeys(
+      'Suggestion Commit Message Input',
+      suggestionCommitMessageInput, 'Commit message');
+    await action.click('Accept Suggestion Button', acceptSuggestionButton);
     await waitFor.invisibilityOf(
       acceptSuggestionButton, 'Suggestion modal takes too long to disappear');
     await waitFor.pageToFullyLoad();
```

Comment:
Below in `this.expectTohaveFeedbackThread`, could you replace the `expect(...)` line with a `waitFor.presenceOf(...)` line? This will have the same effect of failing if the feedbackTabRow is missing, but it will also give the row time to load